pyquil/tests/test_quilt.py

```diff
@@ -195,6 +195,21 @@ def test_program_calibrate():
     assert calibrated == Program('SHIFT-PHASE 0 "rf" -pi').instructions
 
 
+def test_program_calibrate_order():
+    """Test that later calibrations take preference over earlier calibrations."""
```

Comment:
preference -> precedence

Reply:
should also test that in the scenario

1. calibration
2. instruction
3. calibration

calibration 3 is applied to the instruction

Reply:
That is now tested in `test_expand_calibrations`